### PR TITLE
Create README.md in default template

### LINKED ISSUE #5456: README template can be added to forge init
### Component

Forge

### Describe the feature you would like

README.md file can be added to the `forge` template.

This file can contain

```
forge build 
forge test
# to run a single test
forge test --mt <filename> 
```
more info about `anvil` and `chisel` can be also mentioned here.

If this is necessary, I would like to work on this.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by allwin199**
Adding README.md in the default template to give more info on how to work with the provided template.

fixes #5456

### CODE REVIEW

File: cli/assets/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Foundry
+
+Foundry is a smart contract development toolchain.
```
Comment (Evalir) [inline cli/assets/README.md:3]: I'd expand this section to include what the repo readme says about the parts of foundry (forge, cast, anvil)

File: cli/assets/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+### Anvil
+
+```shell
+$ anvil
+```
+
+### Deploy
+
+```shell
+$ forge create --rpc-url <your_rpc_url> --private-key <your_private_key> src/MyContract.sol:MyContract
```
Comment (Evalir) [inline cli/assets/README.md:44]: I'd use `forge script` here instead of `forge create`.

**Review by Evalir — CHANGES_REQUESTED** [review]

File: cli/assets/README.md (hunk 1 above)
Comment (allwin199) [reply cli/assets/README.md:3]:
```
Foundry is a blazing fast, portable and modular toolkit for Ethereum application development written in Rust.

Foundry consists of:

-   Forge: Ethereum testing framework (like Truffle, Hardhat and DappTools).
-   Cast: Swiss army knife for interacting with EVM smart contracts, sending transactions and getting chain data.
-   Anvil: Local Ethereum node, akin to Ganache, Hardhat Network.
-   Chisel: Fast, utilitarian, and verbose solidity REPL.
```

This is what I'm planning to add, let me know your thoughts.

File: cli/assets/README.md (hunk 2 above)
Comment (allwin199) [reply cli/assets/README.md:44]: sure! I will change it to `forge script`

File: cli/assets/README.md (hunk 1 above)
Comment (Evalir) [reply cli/assets/README.md:3]: Looks good!

**Comment by allwin199** [comment]
I have made the suggested changes. 
Also, I have added this file only to [assets directory](https://github.com/foundry-rs/foundry/tree/master/cli/assets).
Let me know if this has to be mapped anywhere to be downloaded with forge init

**Review by Evalir — APPROVED** [review]
lgtm!

**Pull request merged**